### PR TITLE
makes installer create static pod

### PR DESCRIPTION
**Pull request by deads2k**
This now runs!

/assign @sttts

### CODE REVIEW

File: pkg/operator/target_config_reconciler_v311_00.go [hunk 1]
```diff
@@ -90,8 +94,7 @@ func createTargetConfigReconciler_v311_00_to_latest(c TargetConfigReconciler, op
 func manageKubeApiserverConfigMap_v311_00_to_latest(client coreclientv1.ConfigMapsGetter, operatorConfig *v1alpha1.KubeApiserverOperatorConfig) (*corev1.ConfigMap, bool, error) {
 	configMap := resourceread.ReadConfigMapV1OrDie(v311_00_assets.MustAsset("v3.11.0/kube-apiserver/cm.yaml"))
 	defaultConfig := v311_00_assets.MustAsset("v3.11.0/kube-apiserver/defaultconfig.yaml")
-	deploymentOverrides := v311_00_assets.MustAsset("v3.11.0/kube-apiserver/deployment-config-overrides.yaml")
```
Comment (sttts) [inline pkg/operator/target_config_reconciler_v311_00.go]: delete the file as well

File: pkg/operator/target_config_reconciler_v311_00.go [hunk 2]
```diff
@@ -127,19 +130,23 @@ func manageKubeApiserverPublicConfigMap_v311_00_to_latest(client coreclientv1.Co
...
+		required.Spec.Containers[0].ImagePullPolicy = corev1.PullPolicy(operatorConfig.Spec.ImagePullPolicy)
 	case "":
 	default:
-		return nil, false, fmt.Errorf("invalid imagePullPolicy specified: %v", options.Spec.ImagePullPolicy)
+		return nil, false, fmt.Errorf("invalid imagePullPolicy specified: %v", operatorConfig.Spec.ImagePullPolicy)
 	}
-	required.Spec.Containers[0].Image = options.Spec.ImagePullSpec
-	required.Spec.Containers[0].Args = append(required.Spec.Containers[0].Args, fmt.Sprintf("-v=%d", options.Spec.Logging.Level))
+	required.Spec.Containers[0].Image = operatorConfig.Spec.ImagePullSpec
+	// TODO remove
```
Comment (sttts) [inline pkg/operator/target_config_reconciler_v311_00.go]: why hardcoded?

Comment (deads2k) [reply pkg/operator/target_config_reconciler_v311_00.go]: > why hardcoded?

The installer pod has the wrong image and I need to look up how use the downward API

**Comment by deads2k** [comment]
comments addressed.

**Pull request merged**